### PR TITLE
cudf.pandas updates (homepage and landing page)

### PR DESCRIPTION
**Pull request by beckernick**
This PR:
- Adds a brief cudf.pandas on Colab update to the homepage "New" section
- Updates cudf.pandas landing page to indicate GA

### CODE REVIEW

**Comment by beckernick** [comment]
This is now ready for review

File: layouts/page/cudf-pandas.html [hunk 1]
```diff
@@ -159,7 +159,7 @@ <h2><i class="fa-light fa-bookmark"></i> Try Now on Colab</h2>
           {{ $colabImg := ((site.GetPage "_partners/google-colab").Resources.Get "logo.png").Resize "200x" }}
           <a href="https://nvda.ws/rapids-cudf" target="_blank"><img src="{{ $colabImg.RelPermalink }}"></a>
           <p>Take cuDF's new pandas accelerator mode for a test-drive in a free GPU-enabled notebook environment using
-            your Google account by <a href="https://nvda.ws/rapids-cudf" target="_blank">launching on CoLab <i
+            your Google account by <a href="https://nvda.ws/rapids-cudf" target="_blank">launching on Colab <i
```
Comment (exactlyallan) [inline layouts/page/cudf-pandas.html:162]: I would update the link to the blog for consistency. Would change wording to reflect it as well.

**Review by exactlyallan — CHANGES_REQUESTED** [review]
Verify colab notebook links.

**Comment by beckernick** [comment]
Discussed offline -- just updated the homepage Try on Colab links to better differentiate them.

**Review by exactlyallan — APPROVED** [review]

**Pull request merged**